pymobiledevice3/services/dvt/instruments/application_listing.py

```diff
@@ -1,10 +1,16 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pymobiledevice3.services.dvt.dvt_secure_socket_proxy import DvtSecureSocketProxyService
```

Comment:
If you need to import it, import it - don't conditionally import it

Reply:
k